### DISCUSSION TITLE
Error running openupgrade V14

### DISCUSSION THREAD

**Discussion by carlosajh** [OCA/OpenUpgrade#3623] [Q&A]
Hello, I running upgrade from Odoo V13  to V14 with openupgrade and when I run the script i reveceive this error

File "/usr/local/lib/python3.6/dist-packages/openupgradelib/openupgrade.py", line 88, in <module> Many2many = core.fields.Many2many AttributeError: module 'odoo' has no attribute 'fields'

Ubuntu 18.04.1 LTS (GNU/Linux 4.15.0-196-generic x86_64)

Somebody can help me. I am not a developer but for me it is important to be able to update my system.

**Comment by legalsylvain** [comment]
Is your Odoo V14 instance working correctly ? (without openupgrade I mean).
Your error module 'odoo' has no attribute 'fields' looks like something is wrong in your V14 installation.

regards.
